src/action/mod.rs

```diff
@@ -298,6 +298,10 @@ pub enum ActionError {
     Write(std::path::PathBuf, #[source] std::io::Error),
     #[error("Seek path `{0}`")]
     Seek(std::path::PathBuf, #[source] std::io::Error),
+    #[error("Flushing `{0}`")]
+    Flush(std::path::PathBuf, #[source] std::io::Error),
+    #[error("Set length of `{0}`")]
+    SetLen(std::path::PathBuf, #[source] std::io::Error),
```

Comment:
I know that the actual function is `set_len`, but IMHO "Truncating" would be more descriptive. That said, I'm not overly fussed. 
```suggestion
    #[error("Truncating `{0}`")]
    Truncate(std::path::PathBuf, #[source] std::io::Error),
```

Reply:
Let's do Truncate!